ospf6d/ospf6_proto.h

```diff
@@ -22,6 +22,10 @@
 #ifndef OSPF6_PROTO_H
 #define OSPF6_PROTO_H
 
+
+#include <lib/json.h>
```

Comment:
This should be `"json.h"` or `"lib/json.h"`, I don't think `<>` is correct even if it works

Reply:
fixed.